pkg/ipcache/metadata.go

```diff
@@ -81,6 +81,18 @@ type metadata struct {
 	queuedChangesMU lock.Mutex
 	queuedPrefixes  map[netip.Prefix]struct{}
 
+	// queuedRevision is the "version" of the prefix queue. It is incremented
+	// on every *dequeue*. If injection is successful, then injectedRevision
```

Comment:
It seems weird that the godoc comment for `queuedRevision` describes when `injectedRevision` is updated. Should the second sentence be moved to the `injectedRevision` godoc?

Reply:
Just trying to provide additional context, but I can try and make this clearer.